CONTRIBUTING.md

```diff
@@ -116,6 +116,14 @@ Once you have installed the dependencies, run the server as follows:
 python manage.py runserver
 ```
 
+# Run unit tests
+
+Instructions on how to run all test code:
+
+```py
+cd project && python manage.py test
+```
```

Comment:
We can leave out the change directory command from the command example. In the instructions, just add the line "run the following command from within the `project` directory."
```suggestion
Execute unit tests by running the following command from within the `project` directory.

```py
python manage.py test
```
```